zkevm-framework/bin/assigner/CMakeLists.txt

```diff
@@ -13,12 +13,17 @@ target_include_directories(${TARGET_NAME}
     ${CMAKE_CURRENT_SOURCE_DIR}/include
 )
 
+find_package(actor_zk REQUIRED)
```

Comment:
This package is not used here directly. Probable we need to move it into proof-producer config as a dependency?

Reply:
`crypto3::proof-producer-multi-threaded-lib` contains target `actor::zk`

Reply:
Sure, I see. But, for example, it depends on `crypto3::transpiler` as well, though we don't have to specify it here. I think it should be a responsibility of the package to fetch all its dependencies, not of users of the package.

Reply:
done